optimisation/design/Model_Optimisation_Design.md

```diff
@@ -528,6 +601,7 @@ If model optimisation is to be supported when running compiled MATLAB/Horace fro
 - Can compiled MATLAB call pure Python functions?
 	+ How should such functions be provided to MATLAB? As `module.function` strings, memory addresses, etc.?
 - Does compiled MATLAB have access to builtin MATLAB functions like `feval` and `str2func`?
+	+ Either way it can not run arbitrary m-files! No MATLAB user functions can be called using the Python interface.
 
 
```

Comment:
One very important "gotcha" with the compiled Matlab is:

```python
>>> m.eval('py.dict()')
Warning: Ability to call Python libraries from MATLAB is not available with MATLAB Compiler SDK for Python.
```

(the same code works [e.g. calling python from Matlab from within python] if you have a Matlab license, but not with the compiled version!). 

This error also appears if a newer version of Matlab is used to compile the code - so to have Python user model functions we need to use some sort of C++ or inter-process communications work-around which will be a major pain.

Reply:
PS. I think "eluded" should be "alluded" (wherever it is... I missed the push which added that sentence).